extensions/resource-deployment/src/ui/toolsAndEulaSettingsPage.ts

```diff
@@ -102,7 +102,7 @@ export class ToolsAndEulaPage extends ResourceTypePage {
 			};
 			const installedPathColumn: azdata.TableColumn = {
 				value: loc.discoverPathOrAdditionalInformationText,
-				width: 580
```

Comment:
Can you make description a little smaller by around 10-20 px making sure the kubectl description still fits and then increase the InstallPath length?